src/algebra/opposites.lean

```diff
@@ -192,6 +193,31 @@ variable {α}
 @[simp] lemma unop_smul {R : Type*} [has_scalar R α] (c : R) (a : αᵒᵖ) :
   unop (c • a) = c • unop a := rfl
 
+lemma semiconj_by.op [has_mul α] {a x y : α} (h : semiconj_by a x y) :
+  semiconj_by (op a) (op y) (op x) :=
+begin
+  dunfold semiconj_by,
+  rw [← op_mul, ← op_mul, h.eq]
+end
+lemma semiconj_by.unop [has_mul α] {a x y : αᵒᵖ} (h : semiconj_by a x y) :
+  semiconj_by (unop a) (unop y) (unop x) :=
+begin
+  dunfold semiconj_by,
+  rw [← unop_mul, ← unop_mul, h.eq]
+end
+
+lemma commute.op [has_mul α] {x y : α} (h : commute x y) : commute (op x) (op y) :=
```

Comment:
You could also add a simp-lemma
```lean
@[simp] lemma commute_op [has_mul α] {x y : α} :
  commute (op x) (op y) \iff commute x y :=
```
Note that both versions are useful, one for dot-notation, the other for simp.

Reply:
I did and also generalised to semiconj_by_op.